app/models/child.rb

```diff
@@ -147,13 +147,20 @@ def nebraska_estimated_revenue(filter_date)
   end
 
   def revenue_less_family_fee(filter_date)
-    revenue = attendances.for_month(filter_date).pluck(:earned_revenue).sum - (active_nebraska_approval_amount(filter_date)&.family_fee || 0.0)
-    revenue.negative? ? 0.0 : revenue
+    [(absence_revenue(filter_date) + attendance_revenue(filter_date) - active_nebraska_approval_amount(filter_date)&.family_fee.to_f), 0.0].max
+  end
+
+  def attendance_revenue(filter_date)
+    attendances.non_absences.for_month(filter_date).pluck(:earned_revenue).sum
+  end
+
+  def absence_revenue(filter_date)
+    absences, covid_absences = attendances.absences.for_month(filter_date).partition { |absence| absence.absence == 'absence' }
+    absences.take(5).pluck(:earned_revenue).sum + covid_absences.pluck(:earned_revenue).sum # only five absences are allowed per month in Nebraska
```

Comment:
@katelovescode How are we selecting which 5 to take?  If we don't do this already, can we sort by descending earned_revenue and pluck those 5?  JT had some creative logic to make this happen in the spreadsheet model.